kahpp-spring-autoconfigure/src/test/java/dev/vox/platform/kahpp/integration/KaHPPMockServer.java

```diff
@@ -40,22 +40,20 @@ public static void closeServer() {
     mockServer.close();
   }
 
-  public static void mockHttpResponse(String body, int statusCode) {
-    mockHttpResponse(body, statusCode, null);
+  public static void mockHttpResponse(String path, String body, int statusCode) {
```

Comment:
You could have used here method overloading - keep old _mockHttpResponse(String body, int statusCode)_ method also and call from it a new one with hardcoded "/enrich" path - this way you could avoid a lot of changes in the existing code base.

Reply:
True, but the previous signature didn't have much sense